packages/assert/src/assert.js

```diff
@@ -224,6 +226,7 @@ function assert(flag, optDetails = details`Check failed`) {
  * @param {*} actual The value we received
  * @param {*} expected What we wanted
  * @param {Details} [optDetails] The details to throw
+ * @returns {asserts actual is expected}
```

Comment:
Isn't the contents of returns' argument supposed to be type-like. is `asserts actual is expected` looks like the comment. What does assert return when it doesn't throw?

Reply:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#assertion-functions

It's legitimate syntax but doesn't actually work (doesn't check for equality, checks for membership in a type).  So, I'll remove this instance, but preserve the type of `assert` itself, which is similar but correct.

Reply:
I turned out adding as specific types as I could.